<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostEffectDialog</class>
 <widget class="QDialog" name="PostEffectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PostEffect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="marmosetBloomGroupBox">
     <property name="title">
      <string>Marmoset Bloom</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="sizeLabel">
        <property name="text">
         <string>大小</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="marmosetBloomEnableCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSlider" name="sizeHSlider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="brightnessLabel">
        <property name="text">
         <string>亮度</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="brightnessHSlider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="brightnessLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="sizeLineEdit">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="colorPushButton">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>marmosetBloomEnableCheckBox</zorder>
     <zorder>brightnessLabel</zorder>
     <zorder>brightnessHSlider</zorder>
     <zorder>sizeHSlider</zorder>
     <zorder>sizeLabel</zorder>
     <zorder>brightnessLineEdit</zorder>
     <zorder>sizeLineEdit</zorder>
     <zorder>colorPushButton</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="toneMappingGroupBox">
     <property name="title">
      <string>Tone Mapping</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="toneMappingEnableCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="glowGroupBox">
     <property name="title">
      <string>FXAA(反锯齿)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>由于开启反锯齿会导致Bloom效果出现偏差，所以开启Bloom时会关闭FXAA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="FXAAEnableCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="motionBlurGroupBox">
     <property name="title">
      <string>Motion Blur</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="motionBlurDoubleSpinBox">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelBlur">
        <property name="text">
         <string>Blur</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="motionBlurEnableCheckBox">
        <property name="text">
         <string>enable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
